work_queue/test/network_function.py

```diff
@@ -0,0 +1,147 @@
+#! /usr/bin/env python3
```

Comment:
The name network_function does not communicate what it does. serverless_function, coprocess_function?